### ISSUE TITLE
Successfully build standalone - Use Requires additional packages. Please add comments to README file.

### ISSUE THREAD

**Issue by offternet** [step-/yad#7]
Suggest adding to your README file that standalone is an option for  "./configure enable-standalone"  but such use will require packages (to patch) use on other machines.  Must also have same cpu architechture as compilling system.

Successfully compiled, make, installed  your yad 0.42.81 "Standalone" on debian x64 system. Then copied this yad build to a USB stick and tested it on Debian bookworm system "Q4OS - KDE Plazma" containing no previous yad installation of anykind.

yad 0.42.81 Compiled "Standalone" x64  - file size= 939KB

Additional GTK packages installed on target machine (Less than 10MB on Debian bookworm "this will vary by distribution")

========================================================

**1. Error is icon file** ......  
(yad.0.42.81-standalone:26023): Gtk-WARNING **: 17:04:09.472: Error loading icon from file 'yad':
	Couldn’t recognize the image file format for file “yad”

**Not Resolved:** Hard Coded in Compiled yad.

**2. Error: .** /yad.0.42.81-standalone: error while loading shared libraries: 
    libgtk-x11-2.0.so.0: cannot open shared object file: No such file or directory

**resolved:**  sudo apt-get install libgtk2.0-0

The following additional packages will be installed:   libgail-common libgail18 libgtk2.0-bin libgtk2.0-common

**3. Error: ** (yad:4213): Gtk-WARNING **: 18:18:46.759: Unable to locate theme engine in module_path: "adwaita",

**resolved:**  sudo apt-get install gnome-themes-extra gnome-themes-extra-data

The following additional packages will be installed:  gnome-accessibility-themes

The following NEW packages will be installed:  gnome-accessibility-themes gnome-themes-extra gnome-themes-extra-data

**Comment by step-** [comment]
Hi, thank you for your message, and happy holidays.

The correct option name would be `--enable-standalone` with two leading hyphens. However, "this" yad ([yadL]) does not support that configure option; only "the other" yad ([yadU]) does.

Hmm, I could be wrong but I suspect that an underlying issue here is confusing terminology. [yadU]'s "standalone" build does not mean "portable". Was that your expectation -- to get a portable binary -- when you copied the binary to a USB stick, and tested it on a different Linux OS?  
The error messages you reported point at missing dependencies, which you have resolved by installing the required packages. (I will get to the "yad icon" error in a moment).  
The term "standalone" only means that the program configuration lives in the `yad.conf` file, whereas [yadU]'s default build brings in an additional dependency on gsettings.

Perhaps the word "standalone" can mislead someone into thinking that the binary is portable, like a snap package or an appImage. Not so: a properly installed GTK-2 (or GTK-3) system is still needed. I can't change the terminology of the [yadU] project.

As far as the message 'Couldn't recognize the image file format for file "yad"', sometimes I see it too when I build then run the binary from inside the project directory. Moving the binary to another location is usually enough to get rid of that message. Let me know if this works for you.

In summary, [yadL] doesn't support the `--enable-standalone` configuration setting. Configuration data are saved to file `yad.conf`.  See also <https://github.com/step-/yad/blob/cfe6c7064f3daff80953a09a890537a8e68b8d11/feature-comparison.md?plain=1#L220>
and
<https://github.com/step-/yad/blob/cfe6c7064f3daff80953a09a890537a8e68b8d11/feature-comparison.md?plain=1#L292>.

[yadL]: https://github.com/step-/yad
[yadU]: https://github.com/v1cont/yad

**Comment by offternet** [comment]
I assumed standalone did mean that some of the dependicies were stored in the compiled yad, thanks for correcting me giving me a better understanding.  I will have to correct my online comments on my website about the standalone option. 

Thank You Very Much for your indepth explaination and Have a Great New year.

**Comment by step-** [comment]
> I will have to correct my online comments on my website

I clicked on the yadgui.com link in your profile but it's unreachable (not in CloudFare's DNS records).

You are welcome. Happy New Year to you too.
